### PR TITLE
.NET application update (Account, Authentication & Authorization & Alarms)

### PR DESCRIPTION
**Pull request by omar-srbd**
### Change Description ###

This is the third PR in the process of updating the .NET applications section of Tizen Docs site.

Signed off by: [omar.saeed@samsung.com](mailto:omar.saeed@samsung.com)

### CODE REVIEW

**Comment by safir-srbd** [comment]
Closing and reopening PR to rebuild staging site.

File: docs/application/dotnet/guides/account/oauth.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ The specification defines the following 4 grant types as well as an extensibilit
 
 -   Resource owner password credentials
 
-    The resource owner password credentials (such as username and password) can be used directly as an authorization grant to obtain an access token.
+    The resource owner's password credentials (such as username and password) can be used directly as an authorization grant to obtain an access token.
```
Comment (maliha-srbd) [inline docs/application/dotnet/guides/account/oauth.md:55]:
```suggestion
    The resource owner password credentials (such as username and password) can be used directly as an authorization grant to obtain an access token.
```

File: docs/application/dotnet/guides/account/oauth.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ The specification defines the following 4 grant types as well as an extensibilit
...
 
     Even though this grant type requires direct client access to the resource owner credentials, the resource owner credentials are used for a single request and are exchanged for an access token. This grant type can eliminate the need for the client to store the resource owner credentials for future use, by exchanging the credentials with a long-lived access token or refresh token.
```
Comment (maliha-srbd) [inline docs/application/dotnet/guides/account/oauth.md:57]:
```suggestion
    Even though this grant type requires direct client access to the resource owner credentials, the resource owner credentials are used for a single request, and are exchanged for an access token. This grant type can eliminate the need for the client to store the resource owner credentials for future use, by exchanging the credentials with a long-lived access token or refresh token.
```

File: docs/application/dotnet/guides/account/oauth.md [hunk 3]
```diff
@@ -246,7 +246,7 @@ To obtain the required authorization code or access token:
 
         You can request an access token in a single step without obtaining the authorization code explicitly. For the authorization code grant type, the code is obtained after the authentication and passed to the server to obtain the access token internally. For the implicit, resource owner password credentials, and client credentials grant types, you can obtain the access token directly.
```
Comment (maliha-srbd) [inline docs/application/dotnet/guides/account/oauth.md:247]:
```suggestion
        You can request an access token in a single step without obtaining the authorization code explicitly. For the authorization code grant type, the code is obtained after the authentication, and passed to the server to obtain the access token internally. For the implicit, resource owner password credentials, and client credentials grant types, you can obtain the access token directly.
```

File: docs/application/dotnet/guides/account/overview.md [hunk 4]
```diff
@@ -15,13 +15,13 @@ You can use the following personal data features in your .NET applications:
 
 -   [Authentication and Authorization](authentication.md)
 
-    You can obtain an access token by using the OAuth 2.0 authorization. The OAuth 2.0 authorization framework enables you to obtain limited access to an HTTP service, either on behalf of a resource owner by orchestrating an approval interaction between the resource owner and the HTTP service, or by allowing you to obtain access on your own behalf. You can also use the FIDO Universal Authentication Framework to authenticate users.
+    You can obtain an access token by using the OAuth 2.0 authorization. The OAuth 2.0 authorization framework enables you to obtain limited access to an HTTP service, either on behalf of a resource owner by orchestrating an approval interaction between the resource owner and the HTTP service, or by allowing you to obtain access on your behalf. You can also use the FIDO Universal Authentication Framework to authenticate users.
```
Comment (maliha-srbd) [inline docs/application/dotnet/guides/account/overview.md:18]:
```suggestion
    You can obtain an access token by using the OAuth 2.0 authorization. The OAuth 2.0 authorization framework enables you to obtain limited access to an HTTP service, either on behalf of a resource owner by orchestrating an approval interaction between the resource owner and the HTTP service, or by allowing you to obtain access on your behalf. You can also use the FIDO Universal Authentication Framework to authenticate the users.
```

**Review by maliha-srbd — COMMENTED** [review]
Hi @omar-srbd , please incorporate the suggested changes. Thank you. 

I have included a few more suggestions below, please include them as well:


**In the file:** docs/application/dotnet/guides/account/account.md 

**Line#246:** 2. Add 2 more accounts to the database:
**Suggestion:** 2.  Add 2 more accounts to the database:


**In the file:** docs/application/dotnet/guides/account/app-history.md 

**Line#6:** The main features of the Tizen.Context.AppHistory namespace are:
**Suggestion:** The main features of the Tizen.Context.AppHistory namespace are as follows:


**In the line:** docs/application/dotnet/guides/account/authentication.md 

**Line#4:** The authentication and authorization features introduce how you can obtain access tokens using OAuth 2.0, and authenticate users using the FIDO framework.
**Suggestion:** The authentication and authorization features introduce how you can obtain access tokens using OAuth 2.0, and authenticate the users using the FIDO framework.

**Line#10:** You can use the FIDO Universal Authentication Framework to authenticate users. FIDO covers password-less authentications, such as fingerprint, iris, and voice.
**Suggestion:** You can use the FIDO Universal Authentication Framework to authenticate users. FIDO covers password-less authentications, such as fingerprint, iris, and voice detection.


**In the line:** docs/application/dotnet/guides/account/fido.md 

**Line#5:** The FIDO (Fast IDentity Online) Alliance is an organization formed to address the lack of interoperability among strong authentication devices as well as the problems users face with creating and remembering multiple usernames and passwords.
**Suggestion:** The FIDO (Fast IDentity Online) Alliance is an organization formed to address the lack of interoperability among strong authentication devices, as well as the problems users face with creating and remembering multiple usernames and passwords.

**Line#7:** FIDO covers both password-less authentications (through [FIDO UAF components](#fido_uaf_components)), such as fingerprint, iris, voice, and multi-factor authentication, such as OTP and USB dongle. Tizen currently does not support multi-factor authentication.
**Suggestion:** FIDO covers both password-less authentications (through [FIDO UAF components](#fido_uaf_components)), such as fingerprint, iris, voice, and multifactor authentication, such as OTP and USB dongle. Tizen currently does not support multifactor authentication.

**Line#20:** **Figure: UAF high level architecture**
**Suggestion:** **Figure: UAF high-level architecture**

**Line#72:** The FIDO UAF authenticator abstraction layer provides a uniform API to FIDO clients enabling the use of authenticator-based cryptographic
**Suggestion:** The FIDO UAF authenticator abstraction layer provides a uniform API to FIDO clients, enabling the use of authenticator-based cryptographic


**In the file:** docs/application/dotnet/guides/account/oauth.md 

**Line#31:** The specification defines the following 4 grant types as well as an extensibility mechanism for defining additional types:
**Suggestion:** The specification defines the following 4 grant types, as well as an extensibility mechanism for defining additional types:


**In the file:** docs/application/dotnet/guides/alarm/alarms.md 

**Line#62:** - To set an alarm to launch an application:
**Suggestion:** - To set an alarm to launch an application, follow these steps:


**In the file:** docs/application/dotnet/guides/alarm/overview.md 

**Line#7:** You can use the following alarms features in your .NET applications:
**Suggestion:** You can use the following alarm features in your .NET applications:


**In the file:** docs/application/dotnet/guides/index.md 

**Line#3:** Tizen .NET provides a rich set of interfaces allowing you to build compelling TV, mobile, and wearable (preview) applications which achieve native performance.
**Suggestion:** Tizen .NET provides a rich set of interfaces allowing you to build compelling TV, mobile, and wearable (preview) applications which achieves native performance.

**Line#4:** Tizen.Wearable.CircularUI supports Tizen wearable-specific user interfaces and TizenFX API support Tizen platform-specific functionalities.
**Suggestion:** Tizen.Wearable.CircularUI supports Tizen wearable-specific user interfaces, and TizenFX API supports Tizen platform-specific functionalities.

**Review by maliha-srbd — COMMENTED** [review]
Hi @omar-srbd! After another review and discussing with the author, I'm suggesting these changes, please incorporate them as we all as the one below:

Line# 256: 2. Get the accounts:
Suggestion: 2. Get the accounts

**Pull request merged**